process/smartContract/process.go

```diff
@@ -757,14 +772,19 @@ func (sc *scProcessor) ExecuteBuiltInFunction(
 		return vmcommon.UserError, sc.resolveFailedTransaction(acntSnd, tx, txHash, process.ErrBuiltInFunctionsAreDisabled.Error(), snapshot)
 	}
 
-	vmOutput, err := sc.resolveBuiltInFunctions(acntSnd, acntDst, vmInput)
+	var vmOutput *vmcommon.VMOutput
+	defer func() {
+		sc.vmOutputCacher.HasOrAdd(txHash, vmOutput, 0)
```

Comment:
do not put this in defer. put the put only to the end of it.

Reply:
remove every push from here - the only push you need is on finishSCExecution and in ProcessIfError

Reply:
done.